contrib/format-excel/src/main/java/org/apache/drill/exec/store/excel/ExcelBatchReader.java

```diff
@@ -199,7 +245,7 @@ private void getColumnHeaders(SchemaBuilder builder) {
         i++;
       }
       columnWriters = new ArrayList<>(columnCount);
-
+      metadataColumnWriters = new ArrayList<>();
```

Comment:
This field is initialized here, and after the if-else block, it is initialized with the same value. And `columnWriters` value also.

Reply:
Moved initialization to top of method.